shell/components/form/ColorInput.vue

```diff
@@ -82,6 +96,9 @@ export default {
     class="color-input"
     :class="{[mode]:mode, disabled: isDisabled}"
     :data-testid="componentTestid + '-color-input'"
+    :tabindex="isDisabled ? -1 : 0"
+    @keyup.enter.stop="handleKeyups($event)"
+    @keyup.space.stop="handleKeyups($event)"
```

Comment:
I think that we are capable of chaining modifiers in such a way that this can be handled on one line

```suggestion
    @keyup.enter.space.stop="handleKeyups($event)"
```